file_paths.md

```diff
@@ -121,6 +124,10 @@ when installing puppet-agent 32-bit on 64-bit windows, the
 installation path defaults to `C:\Program Files (x86)\Puppet
 Labs`. The examples below assume 2008/2012 and puppet-agent (64-bit).
 
+The package will create two services `puppet` and `mcollective`
+running as `LocalSystem` by default. It will not create a `puppet`
+user or group.
+
```

Comment:
Good question. For FOSS, they are the same (I think). For PE I think everything was prefixed with `pe-`. /cc @haus ?